src/style.css

```
@@ -0,0 +1,117 @@
+:root {
+  --color-teal: #00acdc;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060b35;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
Don't use * selector for properties other than box-sizing. They apply to all the elements, which is not good for the performance